zerver/views/registration.py

```diff
@@ -299,6 +300,17 @@ def accounts_home_with_realm_str(request, realm_str):
     else:
         return HttpResponseRedirect(reverse('zerver.views.registration.accounts_home'))
 
+def accounts_home_with_invite_key(request, confirmation_key):
+    # type: (HttpRequest, str) -> HttpResponse
+        try:
+            get_object_from_key(confirmation_key)
+        except ConfirmationKeyException as exception:
+            return render_confirmation_key_error(request, exception)
+
+        request.session["multiuse_invited"] = True
```

Comment:
hm, it would be better to just process the confirmation key once, in this function, rather than passing it around.
Can we pass the multiuse_invite_object to accounts_home?